tests/test_preprocessing.py

```diff
@@ -321,6 +325,14 @@ def test_regress_out_constants():
     assert_equal(adata, adata_copy)
 
 
+def test_regress_out_reproducible():
+    adata = pbmc68k_reduced()
+    adata = adata.raw.to_adata()[:200, :200].copy()
+    sc.pp.regress_out(adata, keys=["n_counts", "percent_mito"])
+    tester = np.load(DATA_PATH / "regress_test_small.npy")
```

Comment:
Please add a comment mentioning how this file was generated i.e., the commit of `main` it was made from or something.